src/Text/Pandoc/MIME.hs

```diff
@@ -52,6 +59,7 @@ mimeTypesList :: [(String, MimeType)]
 mimeTypesList = -- List borrowed from happstack-server.
            [("gz","application/x-gzip")
            ,("cabal","application/x-cabal")
+           ,("cpt","image/x-corelphotopaint")
```

Comment:
What is that? It's not even in happstack-server as the comment says. Maybe move it to a separate commit?

Reply:
Yeah, moved it to the second commit... the file extension was in the markdown writer code I removed in that second commit.